hooks/useServicesListsHandler.ts

```diff
@@ -0,0 +1,45 @@
+import { useMemo } from "react"
+import { useQuery } from "react-query"
+import { Column } from "react-table"
+import { getServicesLists } from "../api"
+import { ServicesList } from "../models"
+
+export interface ServicesListsHandler {
+  servicesListsData: object[]
+  servicesListsColumns: Column[]
+  isLoading: boolean
+}
+
+export const useServicesListsHandler = (): ServicesListsHandler => {
+  const { 
+    isLoading,
+    isError,
+    data,
+  } = useQuery<ServicesList[], Error>("servicesLists", () => getServicesLists("Status='Published'"))
```

Comment:
For now, this gets all the data from the Airtable at once. When we get more data, we can see if this becomes a performance issue and look into [server-side pagination](https://buildthis.com/client-side-vs-server-side-pagination/).